vars/managerPipeline.groovy

```diff
@@ -123,7 +123,7 @@ def call(Map pipelineParams) {
                    name: 'scylla_mgmt_pkg')
 
             string(defaultValue: "${pipelineParams.get('target_manager_version', '')}",
-                   description: 'master_latest|3.2|3.1. Only for upgrade test',
+                   description: 'master_latest|3.3|3.2. Only for upgrade test',
```

Comment:
Probably small typo? Must be `3.4`?

Reply:
Yes, makes sense to improve it since the upgrade can be run from the released 3.4 to latest, thanks

Reply:
Fixed